xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -1555,6 +1574,21 @@ extension Config {
                 );
             END;
             """
+        ),
+        SQLMigration(
+            version: Int.from(iso8601String: "2024-01-10T11:59:00")!,
+            sql: """
+            /* Tracks `ActivityEvent`s in a queryable stream */
+            CREATE TABLE activity
+            (
+                id       INTEGER PRIMARY KEY AUTOINCREMENT,
+                category TEXT                           NOT NULL,
+                event    TEXT                           NOT NULL,
+                message  TEXT DEFAULT ''                NOT NULL,
+                metadata TEXT DEFAULT '{}'              NOT NULL,
+                created  TEXT DEFAULT CURRENT_TIMESTAMP NOT NULL
+            );
+            """
```

Comment:
Is it possible to also add a test for these migrations using a fresh in-memory SQL DB? This is a question. I don't recall if I put together the code to make that work.

Reply:
Done!